src/gui/components/wxTitleManagerList.cpp

```diff
@@ -935,6 +941,14 @@ wxString wxTitleManagerList::GetTitleEntryText(const TitleEntry& entry, ItemColu
 		return "";
 		//return wxStringFormat2("{}", entry.format);
 	}
+	case ColumnLocation:
+	{
+		if (_pathToUtf8(entry.path).find("usr/") != std::string::npos ||
+			_pathToUtf8(entry.path).find("sys/") != std::string::npos)
```

Comment:
There's technically an edge case where a user could store their games in a path that also contains "usr/" or "sys/". The following code would fix that by creating a relative path from the user's MLC folder first.
```suggestion
		const auto relative_mlc_path = entry.path.lexically_relative(ActiveSettings::GetMlcPath()).string();
		if (relative_mlc_path.starts_with("usr") || relative_mlc_path.starts_with("sys"))
```
There's also other ways you could address this issue.